tests/test_scene_optimizer.py

```diff
@@ -56,13 +56,13 @@ def test_create_computation_graph(self):
             with Client(cluster):
                 sfm_result = dask.compute(sfm_result_graph)[0]
 
-            self.assertIsInstance(sfm_result, SfmResult)
+            self.assertIsInstance(sfm_result, GtsfmData)
 
             # compare the camera poses
             poses = sfm_result.get_camera_poses()
 
             # get active cameras from largest connected component, may be <len(self.loader)
-            connected_camera_idxs = sfm_result.gtsfm_data.get_valid_camera_indices()
+            connected_camera_idxs = sfm_result.get_valid_camera_indices()
```

Comment:
why do we have `get_valid_camera_indices()` in `SfmResult`?

Reply:
maybe i should rename it -- here `sfm_result` is actually an instance of `GtsfmData`. Does that make sense?

Reply:
Ah got it. Yeah I think renaming is better :)

Reply:
I do think `sfm_result` is the best name for the variable name -- as long as we remember that the SfmResult class is now non-existent, so there would be no confusion there